yesod-core/Yesod/Core/Json.hs

```diff
@@ -31,26 +35,27 @@ module Yesod.Core.Json
     , acceptsJson
     ) where
 
-import Yesod.Core.Handler (HandlerFor, getRequest, invalidArgs, redirect, selectRep, provideRep, rawRequestBody, ProvidedRep, lookupHeader)
+import Control.Monad (liftM)
 import Control.Monad.Trans.Writer (Writer)
-import Data.Monoid (Endo)
-import Yesod.Core.Content (TypedContent)
-import Yesod.Core.Types (reqAccept)
-import Yesod.Core.Class.Yesod (defaultLayout, Yesod)
-import Yesod.Core.Class.Handler
-import Yesod.Core.Widget (WidgetFor)
-import Yesod.Routes.Class
+import Data.Aeson (object, (.=))
 import qualified Data.Aeson as J
 import qualified Data.Aeson.Parser as JP
-import Data.Aeson ((.=), object)
-import Data.Conduit.Attoparsec (sinkParser)
-import Data.Text (pack)
-import qualified Data.Vector as V
+import qualified Data.ByteString.Char8 as B8
 import Data.Conduit
+import Data.Conduit.Attoparsec (sinkParser)
 import Data.Conduit.Lift
-import qualified Data.ByteString.Char8 as B8
 import Data.Maybe (listToMaybe)
-import Control.Monad (liftM)
+import Data.Monoid (Endo)
+import Data.Text (pack)
+import qualified Data.Vector as V
+import Yesod.Core.Class.Handler
+import Yesod.Core.Class.Yesod (Yesod, defaultLayout)
+import Yesod.Core.Content (TypedContent)
+import Yesod.Core.Handler
+  (HandlerFor, ProvidedRep, getRequest, invalidArgs, lookupHeader, provideRep, rawRequestBody, redirect, selectRep)
+import Yesod.Core.Types (reqAccept)
+import Yesod.Core.Widget (WidgetFor)
+import Yesod.Routes.Class
```

Comment:
Woops, looks like my `stylish-haskell` got in here.

Reply:
Would you be able to rebase without these unrelated changes?

Reply:
Yup, I'll pull them out.